.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "SDF"]
 	path = SDF
-	url = ../SDF.git
+	url = git@github.com:Warwick-Plasma/SDF.git
```

Comment:
I am not sure you want to keep this. I have problems with the SDF submodule when using the relative path with my fork of the project.